lib/ignore/ignore.go

```diff
@@ -93,16 +106,24 @@ func (m *Matcher) Parse(r io.Reader, file string) error {
 	return err
 }
 
-func (m *Matcher) Match(file string) (result bool) {
+func (m *Matcher) Ignore(file string) (result bool) {
+	return m.match(file)&Ignore != 0
+}
+
+func (m *Matcher) Nuke(file string) (result bool) {
+	return m.match(file)&Nuke != 0
```

Comment:
Yeah, that was intentional because nukes are ignores as well. But I changed the types now based on your comments, it's more clear now.